source/uwp/Renderer/lib/AdaptiveExecuteActionRenderer.cpp

```diff
@@ -15,13 +15,11 @@ namespace winrt::AdaptiveCards::Rendering::Uwp::implementation
         try
         {
             renderContext.LinkSubmitActionToCard(action, renderArgs);
-            return ::AdaptiveCards::Rendering::Uwp::ActionHelpers::BuildAction(action, renderContext, renderArgs, false);
+            return Helpers::ActionHelpers::BuildAction(action, renderContext, renderArgs, false);
```

Comment:
is this Helpers the same as XamlHelpers?

Reply:
Changed this.